terraform/service/main.tf

```diff
@@ -50,12 +50,24 @@ resource "aws_ecs_task_definition" "ecs_service_definition" {
           value = var.domain
         },
         {
-          name  = "GITHUB_ORG"
-          value = var.github_org
+          name = "UI_SECRET_NAME"
+          value = var.ui_secret_name
         },
         {
-          name = "AWS_SECRET_NAME"
-          value = var.aws_secret_name
+          name = "API_SECRET_NAME"
+          value = var.api_secret_name
+        },
+        {
+          name = "API_BUCKET_NAME"
+          value = var.api_bucket_name
+        },
+        {
+          name = "AWS_ENVIRONMENT"
```

Comment:
> You still have AWS_ENVIRONMENT here. I don't think you actually need this since AWS_ACCOUNT_NAME will be set. You could stop passing through the env variable, in the application logic you already set the ENV variable from the account name: AWS_ENV = os.getenv("AWS_ACCOUNT_NAME")
> 
> And when you run the docker locally you don't pass in the AWS_ENVIRONMENT variable:
> 
> docker run -p 127.0.0.1:8000:8000 -e AWS_SECRET_ACCESS_KEY -e AWS_ACCESS_KEY_ID -e API_BUCKET_NAME -e API_SECRET_NAME -e UI_SECRET_NAME -e AWS_ACCOUNT_NAME -e LOCALHOST tech-audit-tool

Fixed